### PR TITLE
move to grunt-jasmine-chromeapp

### PR DESCRIPTION
**Pull request by trevj**
[Review on Reviewable](https://reviewable.io/reviews/uproxy/uproxy-networking/191)

### CODE REVIEW

**Comment by trevj** [comment]
@soycode maybe you could take a super-quick look at this too?

**Comment by bemasc** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -23,32 +23,27 @@
...
     "freedom-for-chrome": "~0.4.7",
     "freedom-for-firefox": "~0.6.4",
     "freedom-pgp-e2e": "^0.2.3",
-    "fs-extra": "~0.12.0",
     "grunt": "~0.4.4",
     "grunt-browserify": "~3.0.1",
     "grunt-contrib-clean": "~0.6.0",
     "grunt-contrib-copy": "~0.5.0",
     "grunt-contrib-jasmine": "~0.8.2",
+    "grunt-jasmine-chromeapp": "^1.3.0",
```
Comment (agallant) [inline package.json]: Probably worth giving this a go with 1.5.0, and I'm guessing there'll be a 1.6.0 or similar in the next day or so.

**Comment by agallant** [comment]
Change itself LGTM - just a nit that it'd be good to try the "latest and greatest" for grunt-jasmine-chromeapp since we're changing it pretty frequently (mostly just to make sure that the added logic doesn't break how it's processing files for you).

**Comment by trevj** [comment]
Good catch! I actually _am_ testing with 1.5.0, this was just a merge mess-up on my side.

Thanks!

**Comment by agallant** [comment]
:+1:

**Pull request merged**